Userland/Utilities/CMakeLists.txt

```diff
@@ -108,7 +108,7 @@ link_with_unicode_data(js)
 target_link_libraries(keymap LibKeyboard LibMain)
 target_link_libraries(less LibMain)
 target_link_libraries(logout LibMain)
-target_link_libraries(ls LibMain)
+target_link_libraries(ls LibMain LibShell)
```

Comment:
This seems _really_ overkill.
I don't think those `escape_token` functions really depend on any of the Shell internals - it seems more sensible to put this in a separate header.

Reply:
You're right, they don't.
I didn't move them to another library because I thought that escaping for single or double quotes are strongly related to the Shell.
I'm not familiar enough with the system hierarchy to fully understand how to tackle this. Should I create a standalone library like LibEscape, does it have a place in AK or maybe a sub-library in LibShell ?